tensorrt_llm/_torch/modules/fused_moe/fused_moe_trtllm_gen.py

```
@@ -292,7 +306,43 @@ def _to_trtllm_gen_activation_type(self,
+        except (ImportError, ModuleNotFoundError):
+            return False
+        return (hasattr(_core, "trtllm_bf16_moe")
+                and hasattr(_core, "trtllm_bf16_routed_moe"))
+
+    def _is_unquantized_path(self) -> bool:
+        return self.quant_config is None or not self.quant_config.layer_quant_mode.has_any_quant(
+            exclude_kv_cache=True)
+
+    @staticmethod
+    def _supports_flashinfer_bf16_routing_method(
+        routing_method: BaseMoeRoutingMethod, ) -> bool:
```

Flake8 (7.3.0)

[error] continuation line with same indent as next logical line

(E125)